apps/launch/components/MyStakingContracts/MyStakingContracts.tsx

```diff
@@ -0,0 +1,52 @@
+import { WalletOutlined } from '@ant-design/icons';
+import { Card, Flex, Typography } from 'antd';
+import Link from 'next/link';
+import { match } from 'ts-pattern';
+import { useAccount } from 'wagmi';
+
+import { UNICODE_SYMBOLS } from 'libs/util-constants/src';
+
+import { LoginV2 } from '../Login';
+
+const { Title, Paragraph } = Typography;
+
+const ICON_STYLE = { fontSize: '56px', color: '#A3AEBB' };
+
+const ConnectWallet = () => {
+  return (
+    <Flex align="center" vertical gap={16}>
+      <WalletOutlined style={ICON_STYLE} />
+      <Paragraph type="secondary" className="text-center">
+        Connect your wallet to allocate your voting power.
+      </Paragraph>
+      <LoginV2 />
+    </Flex>
+  );
+};
+
+export const MyStakingContracts = () => {
+  const { isConnected: isAccountConnected } = useAccount();
+
+  return (
+    <>
```

Comment:
Can we reduce width here, to ~1000px as in Figma?
I used this for the Contract page on Govern, if that suits

```
const StyledMain = styled.main`
  display: flex;
  flex-direction: column;
  max-width: 800px;
  margin: 0 auto;
`;
```